src/main/resources/static/css/techpoker.css

```diff
@@ -1,20 +1,20 @@
 .poker-card {
     /* Add shadows to create the "card" effect */
-    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    box-shadow: 0 4px 8px 0 rgba(255, 255, 255, 0.1);
```

Comment:
I don't really know why, but the contrast between selected and unselected is rather weak, especially when you click somewhere outside, and the hover shadow disappears.

Reply:
I increased the alpha value so that the frame is more visible again. I set the value equal to the :hover value. So the frame below is always shown, and the other 3 sides are added with a :hover.